Kitodo/src/main/java/org/goobi/mq/processors/CreateNewProcessProcessor.java

```diff
@@ -27,6 +27,9 @@
 import org.apache.logging.log4j.Logger;
 import org.goobi.mq.ActiveMQProcessor;
 import org.goobi.mq.MapMessageObjectReader;
+import org.kitodo.api.ugh.exceptions.PreferencesException;
```

Comment:
Why did you add this and the following two exception classes which are not used?